test/test-versions.js

```diff
@@ -51,16 +51,8 @@ function testVersions(dataFilename) {
           supportStatements.push(supportData[browser]);
         }
 
-        const validBrowserVersionsString =
-          'true, false, null' +
-          (validBrowserVersions[browser].length > 0
-            ? ', ' + validBrowserVersions[browser].join(', ')
-            : '');
-        const validBrowserVersionsTruthy =
-          validBrowserVersions[browser].length > 0
-            ? 'true, ' + validBrowserVersions[browser].join(', ')
-            : 'true';
-
+        const validBrowserVersionsString = 'true, false, null, ' + validBrowserVersions[browser].join(', ');
+        const validBrowserVersionsTruthy = 'true, ' + validBrowserVersions[browser].join(', ');
         for (const statement of supportStatements) {
```

Comment:
The previous commit removed the line above this one:
```suggestion

        for (const statement of supportStatements) {
```